app/styles/network/show.scss

```diff
@@ -28,7 +28,7 @@ div#network-container {
   .ember-view { height: 100%; }
 
   div#details-container > * {
-    margin: 0; padding: 0; border: 0;
+    margin: 0; padding: 0; /*border: 0;*/
```

Comment:
Don't comment, just remove it. I'm sure we know how to put it back if we need it.